tests/Fixer/Semicolon/NoEmptyStatementFixerTest.php

```diff
@@ -548,16 +548,6 @@ public function provideWithShortOpenTagCases()
                 );
         }
 
-        // HHVM parses '<?=' as T_ECHO instead of T_OPEN_TAG_WITH_ECHO
```

Comment:
did hhvm fixed the bug ?

Reply:
Please see the linked comment for that change in the commit message. I removed it due to get the suite green. I can not say whether or not they fixed the bug.

Reply:
@keradus 

the test case is invalid on HHVM and PHP, we just never saw the issue because the tests never run on Travis and the majority of the contributors has `short_open_tag` disabled

```
1) PhpCsFixer\Tests\Fixer\Semicolon\NoEmptyStatementFixerTest::testCasesWithShortOpenTag with data set #1 ('<?= ', '<?= ;')
Failed asserting that 'PHP Parse error: syntax error, unexpected ';' on line 1.\n
\n
Source:\n
<?= ;' is null.
```

```
[12:08 PM]-[possum@zoo1]-[/home/possum/work/PHP-CS-Fixer]-[git master] 
$ php -v
PHP 7.2.3-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Mar  6 2018 11:18:25) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.3-1+ubuntu16.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies

[12:08 PM]-[possum@zoo1]-[/home/possum/work/PHP-CS-Fixer]-[git master] 
$ cat /etc/php/7.2/cli/php.ini|grep short_open_tag
151:; short_open_tag
197:short_open_tag = On
```